sigma/pipelines/netwitness/netwitness.py

```diff
@@ -0,0 +1,292 @@
+"""Module for pySigma NetWitness processing pipelines"""
+
+from typing import List
+
+from sigma.processing.conditions import IncludeFieldCondition
+from sigma.processing.pipeline import ProcessingItem, ProcessingPipeline
+from sigma.processing.transformations import ConvertTypeTransformation
+
+from sigma.backends.netwitness.transformations import UnquoteStringTransformation
+
+field_transformations_to_string: List[str] = [
```

Comment:
These variables seem like configuration items to me.

Reply:
Same here

Reply:
Same as above